proxy/envoy/config.go

```diff
@@ -112,6 +118,15 @@ func buildConfig(listeners Listeners, clusters Clusters, mesh *proxyconfig.Proxy
 		StatsdUDPIPAddress: mesh.StatsdUdpAddress,
 	}
 
+	if lds {
```

Comment:
Not yet. I think Istio-on-Istio might need to skip LDS. A proxy in front of pilot might need to be statically configured.